tests/admin_views/test_skip_link_to_content.py

```diff
@@ -0,0 +1,148 @@
+        skip_link = self.selenium.find_element(By.CLASS_NAME, "skip-to-content-link")
+        self.assertTrue(skip_link.is_displayed())
+        ActionChains(self.selenium).send_keys(Keys.RETURN, Keys.TAB).perform()
+        actors_add_url = reverse("admin:admin_views_actor_add")
+        actors_a_tag = self.selenium.find_element(
+            By.CSS_SELECTOR, f"#content [href='{actors_add_url}']"
+        )
+        self.assertEqual(self.selenium.switch_to.active_element, actors_a_tag)
+
+        # Go to the Actor form and the first input will be focused automatically.
+        with self.wait_page_loaded():
+            actors_a_tag.send_keys(Keys.RETURN)
+        first_input = self.selenium.find_element(By.ID, "id_name")
+        self.assertEqual(self.selenium.switch_to.active_element, first_input)
+
+    def test_dont_use_skip_link_to_content(self):
+        from selenium.webdriver.common.by import By
+
+        self.admin_login(
+            username="super",
+            password="secret",
+            login_url=reverse("admin:index"),
+        )
+        self.selenium.get(self.live_server_url + reverse("admin:index"))
+
+        # `Skip link` is not present.
+        skip_link = self.selenium.find_element(By.CLASS_NAME, "skip-to-content-link")
+        self.assertFalse(skip_link.is_displayed())
+
+        # 1st TAB is pressed, `skip link` is shown.
+        body = self.selenium.find_element(By.TAG_NAME, "body")
+        body.send_keys(Keys.TAB)
+        self.assertTrue(skip_link.is_displayed())
+
+        # The 2nd TAB will focus the page title.
+        body.send_keys(Keys.TAB)
+        django_administration_title = self.selenium.find_element(
+            By.LINK_TEXT, "Django administration"
+        )
+        self.assertFalse(skip_link.is_displayed())  # `skip link` disappear.
+        self.assertEqual(
+            self.selenium.switch_to.active_element, django_administration_title
+        )
+
+    def test_skip_link_is_skipped_when_there_is_searchbar(self):
+        from selenium.webdriver.common.by import By
+
+        self.admin_login(
+            username="super",
+            password="secret",
+            login_url=reverse("admin:index"),
+        )
+        self.selenium.get(self.live_server_url + reverse("admin:index"))
+
+        group_a_tag = self.selenium.find_element(By.LINK_TEXT, "Groups")
+        with self.wait_page_loaded():
+            group_a_tag.click()
+
+        # `Skip link` is not present.
+        skip_link = self.selenium.find_element(By.CLASS_NAME, "skip-to-content-link")
+        self.assertFalse(skip_link.is_displayed())
+
+        # `Searchbar` has autofocus.
+        searchbar = self.selenium.find_element(By.ID, "searchbar")
+        self.assertEqual(self.selenium.switch_to.active_element, searchbar)
+
+    def test_skip_link_with_RTL_language_doesnt_create_horizontal_scrolling(self):
+        from selenium.webdriver.common.by import By
+
+        with override_settings(LANGUAGE_CODE="ar"):
```

Comment:
I just tested with `Arabic`.

We could test it over all RTL languages if it is needed!
 https://github.com/django/django/blob/e59d1ff5627cf9cc551e160d65858a37693de4ed/django/conf/global_settings.py#L157

Reply:
Unless there's a significant difference, a single RTL language seems fine... 🤔